src/blocks/blocks.h

```diff
@@ -1,16 +1,11 @@
 #ifndef SRC_BLOCKS_BLOCKS_H_
 #define SRC_BLOCKS_BLOCKS_H_
 
-#ifdef __cplusplus
-extern "C" {
-#endif
+#include "cryptonote_config.h"
 
-const unsigned char *get_blocks_dat_start(int testnet, int stagenet);
-size_t get_blocks_dat_size(int testnet, int stagenet);
-
-#ifdef __cplusplus
+namespace Blocks
+{
+    const unsigned char *GetCheckpointsData(cryptonote::network_type network, size_t &size);
```

Comment:
How about returning a `epee::span<const unsigned char>`? Both the values being returned could be grouped together; the out parameter could be dropped.

Reply:
Right, but will introduce `epee` dependency. I would like to keep it that way, let me know your thoughts.

Reply:
Span is header only, so there is no lib dependency. And the header only pulls in C++ headers. The file could be outside of epee, but is used for various hex stuff in epee.

Reply:
Resolved